libraries/botbuilder-dialogs-adaptive-testing/src/httpRequestMocks/httpRequestSequenceMock.ts

```diff
@@ -81,8 +81,12 @@ export class HttpRequestSequenceMock extends HttpRequestMock implements HttpRequ
         }
         const response = new SequenceResponseManager(this.responses);
         const url = parse(this.url);
-        let path = this.url.substr(url.origin.length);
+        let path: string | RegExp = this.url.substr(url.origin.length);
         path = path.startsWith('/') ? path : '/' + path;
+        if (path.includes('*')) {
+            // eslint-disable-next-line security/detect-non-literal-regexp
```

Comment:
Note: [This security check is for DoS attacks](https://github.com/nodesecurity/eslint-plugin-security/blob/master/docs/regular-expression-dos-and-node.md), which I don't believe we'll need to be too concerned about in a testing library.